tutorial/developing_packages/workspaces.rst

```diff
@@ -0,0 +1,285 @@
+.. _tutorial_workspaces:
+
+Workspaces (incubating)
```

Comment:
Are we sure this doesn't generate an external link that includes ``https:/..../workspaces#Workspaces(incubating)`` in the link, so it will be broken when we change it? 
Moving the docs here was the main reason, to be able to start providing permanent links that won't break in the future.

Reply:
I think with the incubating warning down below is enough, may we remove the `(incubating)` ?